### DISCUSSION TITLE
[Form] Can validates TagsInput tag?

### DISCUSSION THREAD

**Discussion by ZerglingGo** [filamentphp/filament#6738] [Help] [answered]
```php
Forms\Components\TagsInput::make('sources')->label('IP Address')->ip(),
```

I designed multiple ip address input form with `TagsInput`.
`sources` property saves array into database

When validation `TagsInput`, it failed with not valid ip address

Is it possible validate separately for each tag?

**Comment by danharrin** [comment, marked as answer]
Try `nestedRecursiveRules(['ip'])`

**Reply by ZerglingGo** [reply]
Thank you, It works fine!
